apps/admin-gui/src/assets/config/defaultConfig.json

```diff
@@ -93,6 +93,16 @@
+            "link_en": "https://perun-aai.org/",
+            "label_en": ""
```

Comment:
I don't think that by default we want for all instances to show these logos. This shouldn't be part of the default config.

Reply:
this was just for the review, I will remove it once the review is done

Reply:
Things like this might get accidentally merged. It would be better to put the logo in a comment to this PR.